<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidgetClass</class>
 <widget class="QMainWindow" name="SettingsWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SettingsWidget</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>331</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="10" column="1">
      <widget class="QLabel" name="gameboardSizeLabel">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Gameboard size</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="timeLimitLabel">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Time limit for each player</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QSpinBox" name="gameboardSize">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>6</number>
       </property>
       <property name="maximum">
        <number>35</number>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="maxDotsLabel">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Maximum dots (pins) per player</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QSpinBox" name="maxDots">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>35</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="gamemodeLabel">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Gamemode</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="12" column="1">
      <widget class="QLabel" name="maxBridgesLabel">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Maximum bridges per player</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="timeLimit">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>30</number>
       </property>
       <property name="maximum">
        <number>1200</number>
       </property>
       <property name="value">
        <number>180</number>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QSpinBox" name="maxBridges">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>35</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QComboBox" name="gamemodeOption">
       <item>
        <property name="text">
         <string>Default</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bulldozer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mines</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>-170</x>
      <y>70</y>
      <width>679</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="backToMenuButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>480</y>
      <width>225</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Go back to main menu</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>150</y>
      <width>331</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="2" column="2">
      <widget class="QLineEdit" name="player1Nickname">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maxLength">
        <number>10</number>
       </property>
       <property name="placeholderText">
        <string>Player1</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="player2Label">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Player2's nickname</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="player2Nickname">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maxLength">
        <number>10</number>
       </property>
       <property name="placeholderText">
        <string>Player2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="player1Label">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Player1's nickname</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="timeLimitLabel_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>130</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>in seconds</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>player1Nickname</tabstop>
  <tabstop>player2Nickname</tabstop>
  <tabstop>timeLimit</tabstop>
  <tabstop>gamemodeOption</tabstop>
  <tabstop>gameboardSize</tabstop>
  <tabstop>maxDots</tabstop>
  <tabstop>maxBridges</tabstop>
  <tabstop>backToMenuButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
